interactive_templates/templates/v2/tests/test_event_counts.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+from analysis.event_counts import get_summary_stats, round_to_nearest
+from hypothesis import given
+from hypothesis import strategies as st
+
+
+@given(
+    st.integers(max_value=100_000_000, min_value=0), st.integers(min_value=1)
+)  # Generate random integers for x and base (base > 0)
+# Possible values for x are clamped. For large numbers the test will fail due to the limited precision of
+# floating-point arithmetic
```

Comment:
I don't understand this comment. Do you mean possible values of `x` are within the interval `[0, 100,000,000]`? Are large numbers within this interval or larger than this interval?

Reply:
It was failing for much larger numbers, numbers this function wouldn't ever be given. I'll make that clearer.